### PR TITLE
Add TypeScript Books

### PR DESCRIPTION
**Pull request by tomriddle7**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by Thenlie — APPROVED** [review]
These look good to me. All of the links seem useful and relevant. My only suggestion would be to squash these commits, which can be done on merge if nothing else.

File: books/free-programming-books-ko.md [hunk 1]
```diff
@@ -237,6 +238,14 @@
 * [Springboot 2.X 정리](https://djunnni.gitbook.io/springboot) - 이동준
 
 
+### TypeScript
+
+* [5분 안에 보는 타입스크립트](https://typescript-kr.github.io)
+* [타입스크립트 핸드북](https://joshua1988.github.io/ts)
+* [한눈에 보는 타입스크립트](https://heropy.blog/2020/01/27/typescript)
```
Comment (eshellman) [inline books/free-programming-books-ko.md:245]:
```suggestion
* [한눈에 보는 타입스크립트](https://heropy.blog/2020/01/27/typescript) - HEROPY
Tech
```

File: books/free-programming-books-ko.md [hunk 2]
```diff
@@ -237,6 +238,14 @@
 * [Springboot 2.X 정리](https://djunnni.gitbook.io/springboot) - 이동준
 
 
+### TypeScript
+
+* [5분 안에 보는 타입스크립트](https://typescript-kr.github.io)
+* [타입스크립트 핸드북](https://joshua1988.github.io/ts)
```
Comment (eshellman) [inline books/free-programming-books-ko.md:244]:
```suggestion
* [타입스크립트 핸드북](https://joshua1988.github.io/ts) - Captain Pangyo
```

**Review by eshellman — CHANGES_REQUESTED** [review]
Please add author names where appropriate.

**Comment by eshellman** [comment]
Thanks!

**Pull request merged**